spec/toplevelmenu/checklist.txt

```diff
@@ -0,0 +1,59 @@
+Description block & feature	Result
+
+TopLevelMenu	////////////
+"Verify that user can hover on the available
+tabs on the TopLevelMenu"	Pass
+"Verify that user can click on the available
+tabs on the TopLevelMenu"	Pass
+"Verify that user can click on the subtabs 
+on the Women tab"	Pass
+"Verify that user can click on the subtabs 
+on the Dresses tab"	Pass
+
+Sidebar (Filter)	////////////
+Verify that 'Tops' filter tab is opens	Pass
+Verify that 'Dresses' filter tab is open	Pass
+Verify that 'Tops' filter tab select	Pass
+Verify that 'Dresses' tab filter select	Pass
+Verify that 'Size' filter tab is select	Pass
+Verify that 'Color' filter tab is select	Pass
+Verify that 'Compositions' filter tab is select	Pass
+Verify that 'Styles' filter tab is select	Pass
+Verify that 'Properties' filter tab is select	Pass
+Verify that 'Availability' filter tab is select	Pass
+Verify that 'Manufacturer' filter tab is select	Pass
+Verify that 'Condition' filter tab is select	Pass
+ Verify that user can edit 'Price' by range filter	Pass
+Verify that user can sort by dropdown list	Pass
+
+Items	////////////
+"Verify that user can click on the any
+available item on the main page (click
+on the icon)"	Pass
+"Verify that user can click on the any
+available item on the main page (click
+on the name)"	Pass
+"Verify that user can change color of the
+selected item"	Pass
+"Verify that user can click on the photo and see
+the current item"	Pass
+Verify that user can switch the photos 	Pass
+"Verify that user can swith quantity by using
+arrow buttons"	Pass
+Verify that user can swith quantity by typing valid data (numbers)	Pass
+Verify that user can swith quantity by typing invalid data (numbers)	Pass
```

Comment:
What invalid data is? bad format, numbers below zero, special symbols. More precision pls